04/flight-service/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+quarkus.http.port=8079
+quarkus.http.test-port=8085
+
+quarkus.grpc.server.port=9005
+
+quarkus.grpc.clients.passenger-service.port=9000
+quarkus.grpc.clients.passenger-service.host=localhost
+
+
+# configure your datasource
+#quarkus.datasource.db-kind = postgresql
+#quarkus.datasource.username = user
+#quarkus.datasource.password = password
+#quarkus.datasource.reactive.url = vertx-reactive:postgresql://localhost:5432/flight-database
```

Comment:
Config profiles? Not sure if you covered that, but everything except for the URL can be there all the time, and URL just moved to prod config profile.

Reply:
It's mentioned in the first lecture. I'll remove this from this lecture